src/main/java/org/eclipsefoundation/git/eca/model/ValidationRequest.java

```diff
@@ -21,21 +22,21 @@
  *
  */
 public class ValidationRequest {
-	private String repoUrl;
+	private URL repoUrl;
 	private List<Commit> commits;
 	private ProviderType provider;
 
 	/**
 	 * @return the repoUrl
 	 */
-	public String getRepoUrl() {
+	public URL getRepoUrl() {
```

Comment:
URL can trigger DNS resolution in sneaky ways. Use the much safer URI instead.

Reply:
I'll test that now, thanks!